code/go/internal/validator/test/packages/input_template/agent/input/template.yml.hbs

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
nit: It would be nice to put a real content in at least one file, so this `template.yml.hbs` won't be so mysterious.

Reply:
@jalvz Can you open a PR with this? Because it is exactly what I'm looking for to have an example.

@mtojek These test packages are great. Example and testing in one go.

Reply:
BTW, you could even put a short APM Example in here to make it more concrete.

Reply:
@ruflin I added https://github.com/elastic/package-spec/commit/76738747c9a03a48a926dbacf655f359162b2557, is not that what you were looking for?
I don't think I can use this file as a real hbs example because it is referenced in the test (correct me if Im wrong)

Also didn't want to add an APM-like config because this can be anything, it has exactly the same structure as the stream templates.

Reply:
Got it. I missed that you added two template files. The part I stumble over is that you used `group`. This is just an example I assume. It can all be on the top level like `foo: bar`?

If we could use apm example here, @mtojek will know best. My preference is always to have a real example if possible.